README.md

```diff
@@ -6,6 +6,16 @@
 [![Known Vulnerabilities](https://snyk.io/test/github/UserOfficeProject/user-office-core/master/badge.svg?targetFile=package.json)](https://snyk.io/test/github/UserOfficeProject/user-office-core/master?targetFile=package.json)
 
 ---
+## Contributing
+
+If you're planning to contribute to this project by adding new functionality, we encourage you to discuss it first in the Discussions tab. This helps ensure that your proposed changes align with the project's goals and prevents duplicate efforts. Here's how you can initiate a discussion:
```

Comment:
Is it only new functionality we want to be contributed? If they spotted a bug or something similar would they just create new issue in the repo?

Reply:
The idea is to have a converstaion about any new additions someone wants to make before jumping into action. So, if you find a bug or something similar, go ahead and open an issue for it. But if you're considering adding a new feature, it's best to start a discussion first in the Discussions tab.